### PR TITLE
Added typescript resource for the issue 919

### PR DESCRIPTION
**Pull request by ursmahi**
## Fixes Issue
Closes [#919 ](https://github.com/rupali-codes/LinksHub/issues/919)

## Changes proposed
I have added  the new resource in typescript language section.

## Screenshots
typescript_pull

### CODE REVIEW

File: database/languages/typescript.json [hunk 1]
```diff
@@ -19,5 +19,12 @@
     "url": "https://basarat.gitbook.io/typescript/",
     "category": "languages",
     "subcategory": "typescript"
+  },
+  {
+    "name": "Build a TypeScript Project From Scratch",
+    "description": "Resource for learning TypeScript from scratch to level-up skills. End-to-end experience of taking from a blank editor to battle-tested production code. A hands-on tutorial where we build a library from scratch, share it with the world, and consume it in a production application.",
```
Comment (CBID2) [inline database/languages/typescript.json:25]: [Proofreading]
```suggestion
    "description": "If you want to learn TypeScript from scratch, this resource is for you. You will have the experience of turning a  blank editor into battle-tested production code. This source also provides a hands-on tutorial where we build a library from scratch, share it with the world.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @ursmahi! :) I pointed out a proofreading error in my inline comment. In the future, consider running your descriptions through Grammarly or your word processing software's spell check before making a pull request. Other than that, you're off to a great start! :)

**Review by ujjawaltyagii — APPROVED** [review]
lgtm

**Review by rupali-codes — APPROVED** [review]
lgtm too

**Pull request merged**